multicluster/config/overlays/leader-ns/coverage/manager_command_patch_coverage.yaml

```diff
@@ -7,7 +7,11 @@ spec:
   template:
     spec:
       containers:
-        - command: ["/bin/sh"]
-          args: ["-c", "/antrea-mc-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-mc-controller.cov.out leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
-          name: antrea-mc-controller
+        - args:
+          - antrea-mc-controller-coverage
```

Comment:
I don't think it will work without the leading slash, it won't find the directory. Maybe that's why the MC controller fails to deploy in Jenkins?